Makefile

```diff
@@ -33,69 +38,77 @@ help:
 	@echo "  dumpdb           Create a postgres database dump with timestamp used as file name"
 	@echo "  loaddb           Load a database dump into postgres, file name in DB_DUMP_FILE"
-	@echo "  build-frontend   Builds the frontend static files"
-	@echo "  build-frontend-w Watches the frontend static files and builds on change"
+	@echo "  build-tagadmin   Builds the tag_admin frontend static files"
+	@echo "  build-tagadmin-w Watches the tag_admin frontend static files and builds on change"
 	@echo "  sync-projects    Runs the synchronization task on all projects"
 	@echo "  requirements     Compiles all requirements files with pip-compile\n"
 
-.docker-build:
-	make build
+.frontend-build:
+	make build-frontend
+.server-build:
+	make build-server
 
-build:
-	cp ./docker/config/webapp.env.template ./docker/config/webapp.env
-	sed -i -e 's/#SITE_URL#/$(subst /,\/,${SITE_URL})/g' ./docker/config/webapp.env
+build: build-frontend build-server
+build-frontend: server-env
+	"${DC}" build frontend
+	touch .frontend-build
+build-server: server-env
+	"${DC}" build --build-arg USER_ID=$(USER_ID) --build-arg GROUP_ID=$(GROUP_ID) server
+	touch .server-build
 
-	"${DC}" build --build-arg USER_ID=$(USER_ID) --build-arg GROUP_ID=$(GROUP_ID) webapp
+server-env:
+	cp ./docker/config/server.env.template ./docker/config/server.env
+	sed -i -e 's/#FRONTEND_URL#/$(subst /,\/,${FRONTEND_URL})/g;s/#SITE_URL#/$(subst /,\/,${SITE_URL})/g' ./docker/config/server.env
 
-	touch .docker-build
+setup: .server-build
+	"${DC}" run server //app/docker/server_setup.sh
 
-setup: .docker-build
-	"${DC}" run webapp //app/docker/set_up_webapp.sh
-
-run: .docker-build
-	"${DC}" run --rm --service-ports webapp
+run: .frontend-build .server-build
+	"${DC}" up
 
 clean:
-	rm .docker-build
+	rm -f .docker-build .frontend-build .server-build
+
+test: test-server test-frontend
 
-test:
-	"${DC}" run --rm webapp //app/docker/run_tests.sh
+test-server:
+	"${DC}" run --rm server //app/docker/server_tests.sh
 
 test-frontend: jest
 jest:
-	"${DC}" run --rm -w //app/frontend webapp yarn test
+	"${DC}" run --rm -w //frontend frontend yarn test
 
 pytest:
-	"${DC}" run ${run_opts} --rm webapp pytest --cov-report=xml:pontoon/coverage.xml --cov=. $(opts)
+	"${DC}" run ${run_opts} --rm server pytest --cov-report=xml:pontoon/coverage.xml --cov=. $(opts)
 
 flake8:
-	"${DC}" run --rm webapp flake8 pontoon/
+	"${DC}" run --rm server flake8 pontoon/
 
 black:
-	"${DC}" run --rm webapp black pontoon/
+	"${DC}" run --rm server black pontoon/
 
 pyupgrade:
-	"${DC}" run --rm webapp pyupgrade --exit-zero-even-if-changed --py38-plus *.py `find pontoon -name \*.py`
+	"${DC}" run --rm server pyupgrade --exit-zero-even-if-changed --py38-plus *.py `find pontoon -name \*.py`
 
 types:
-	"${DC}" run --rm -w //app/frontend webapp yarn types
+	"${DC}" run --rm -w //frontend frontend yarn types
 
 prettier:
-	"${DC}" run --rm webapp npm run prettier
+	"${DC}" run --rm server npm run prettier
```

Comment:
How do I run `prettier` (and `check-prettier` and `eslint`) in the frontend container?

Reply:
I added the frontend commands to the mentioned tasks.